helm/telemetry-module/values.yaml

```diff
@@ -0,0 +1,51 @@
+experimental:
+  enabled: true
+regular:
+  enabled: false
+manager:
+  container:
+    args:
+      - --cert-dir=/tmp
+      - --high-priority-class-name=telemetry-priority-class-high
+      - --normal-priority-class-name=telemetry-priority-class
+    containerSecurityContext:
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+          - ALL
+      privileged: false
+      readOnlyRootFilesystem: false
+    env:
+      alpineImage: europe-docker.pkg.dev/kyma-project/prod/external/library/alpine:3.22.1
+      appLogLevel: info
+      fluentBitExporterImage: europe-docker.pkg.dev/kyma-project/prod/directory-size-exporter:v20250724-d99b68f4
+      fluentBitImage: europe-docker.pkg.dev/kyma-project/prod/external/fluent/fluent-bit:4.0.8
+      gomemlimit: 300MiB
+      otelCollectorImage: europe-docker.pkg.dev/kyma-project/prod/kyma-otel-collector:0.132.0-main
+      selfMonitorImage: europe-docker.pkg.dev/kyma-project/prod/tpi/telemetry-self-monitor:3.5.0-8d9d348
+    image:
+      repository: europe-docker.pkg.dev/kyma-project/prod/telemetry-manager:main
+    resources:
+      limits:
+        memory: 384Mi
+      requests:
+        cpu: 5m
+        memory: 50Mi
+  podSecurityContext:
+    runAsNonRoot: true
+    seccompProfile:
+      type: RuntimeDefault
+  serviceAccount:
```

Comment:
why is this needed ?

Reply:
not more required, removed from values.yaml

Reply:
Sorry for confusion I meant the serviceAccount and empty annotations